pkg/github/datasource.go

```diff
@@ -136,7 +136,13 @@ func NewDatasource(ctx context.Context, settings models.Settings) *Datasource {
 
 	httpClient := oauth2.NewClient(ctx, src)
 
-	return &Datasource{
-		client: githubv4.NewClient(httpClient),
+	if settings.GithubUrl == "" {
+		return &Datasource{
+			client: githubv4.NewClient(httpClient),
+		}
+	} else {
+		return &Datasource{
+			client: githubv4.NewEnterpriseClient(settings.GithubUrl + "/api/graphql", httpClient),
+		}
 	}
```

Comment:
```suggestion
	if settings.GithubUrl == "" {
		return &Datasource{
			client: githubv4.NewClient(httpClient),
		}
	} 
	return &Datasource{
		client: githubv4.NewEnterpriseClient(fmt.Sprintf("%s/api/graphql", settings.GithubUrl), httpClient),
		}
	}
```

The else here isn't necessary since we're doing an early return in the if statement.

Also, I try to avoid string concatenation wherever possible in favor of `fmt.Sprintf`.

Reply:
fixed